### PR TITLE
Add `Carthage` and `SPM` to `README`

### PR DESCRIPTION
**Pull request by adimiz1**
### Brief Summary of Changes
I've updated the README.md file and the .gitingore file.

#### What does this PR address?
- [ ] GitHub issue (Add reference - #XX)
- [x] Refactoring
- [ ] New feature
- [ ] Bug fix
- [ ] Adds more tests

#### Are tests included?
- [ ] Yes
- [x] No

#### Reviewer, please note:
Nothing major, updated the Readme.md to contain installation instructions for carthage (which I personally verified to work)
And added OSX files to the .gitignore file so they wont slip into the repo.

#### Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation. (I changed the documentation)
- [ ] I ran the full test suite before pushing the changes and all the tests pass.

### CODE REVIEW

**Review by asisayag2 — APPROVED** [review]
Kululu🍾

File: README.md [hunk 1]
```diff
@@ -62,6 +62,23 @@ Then, run the command:
...
+```
+
+Then, run the command:
+
+```bash
+carthage update --use-xcframeworks
+```
+
+### Swift Package Manager
+    * File > Add Pakcages... >
```
Comment (const-cloudinary) [inline README.md:78]:
```suggestion
    * File > Add Packages... >
```

File: README.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ platform :ios, '9.0'
 use_frameworks!
 
 target 'MyApp' do
-  pod 'Cloudinary', '~> 3.0'
+  pod 'Cloudinary', '~> 3.1.0'
```
Comment (const-cloudinary) [inline README.md:55]: We don't want to update this number on every new(minor/patch) version released, let's reconsider this.

File: .gitignore [hunk 3]
```diff
@@ -2,6 +2,8 @@
 #
 # gitignore contributors: remember to update Global/Xcode.gitignore, Objective-C.gitignore & Swift.gitignore
 
+*.DS_Store
```
Comment (const-cloudinary) [inline .gitignore:5]: Please include this in a separate PR, this change is not related to "Carthage readme" update

**Review by const-cloudinary — CHANGES_REQUESTED** [review]

**Comment by adimiz1** [comment]
1. Fixed type on Packages
2. Fixed version number back to 3.0 in both CocoaPods and Carthage
3. Removed the .gitignore change into anohter branch.

File: README.md [hunk 4]
```diff
@@ -62,6 +62,23 @@ Then, run the command:
 pod install
 ```
 
+### Carthage
+
```
Comment (const-cloudinary) [inline README.md:66]: Please add a brief description

**Pull request merged**